src/core/feedforward.jl

```diff
@@ -717,6 +717,7 @@ function feedforward_update!(
             IS.@record :simulation FeedForwardUpdateEvent(
                 "FeedForward",
                 current_time,
+                1,  # TODO DT
```

Comment:
Can you clarify what the `TODO` is over here?

Reply:
This needs to be removed. I initially had an idea of tracking time step as an integer. I'll fix all instances.